packages/landing/app/page.tsx

```diff
@@ -5,54 +5,101 @@ import { STACKLY_APP_URL } from "@/constants";
-            src="/assets/images/landing-amount-widget.png"
-            height={200}
-            width={512}
-          />
-        </Link>
-        <div className="absolute w-full -top-36 -z-10 h-[460px] bg-radial-gradient"></div>
-      </div>
-      <div className="mx-auto flex items-center px-5 py-2 bg-black/5 rounded-[20px] w-fit space-x-6 mt-20">
-        <div className="flex items-center space-x-2">
-          <BodyText size={3} weight="medium" className="text-em-med">
-            Total Stacks created:
-          </BodyText>
-          <BodyText size={3} weight="bold">
-            732
-          </BodyText>
+    <main className="mt-14">
+      <section className="max-w-6xl px-6 mx-auto">
+        <div className="space-y-6 text-center ">
+          <DisplayText>Stack crypto over time</DisplayText>
+          <HeadingText className="!font-medium text-em-med max-w-2xl mx-auto">
+            Stackly is a simple, non-custodial tool that does recurring buys of
+            a token at a choosen frequency. Making it easy to DCA.
+          </HeadingText>
         </div>
-        <div className="flex items-center space-x-2">
-          <BodyText size={3} weight="medium" className="text-em-med">
-            Total transactions:
-          </BodyText>
-          <BodyText size={3} weight="bold">
-            1232
-          </BodyText>
+        <ButtonLink
+          target="_blank"
+          href={STACKLY_APP_URL}
+          size="lg"
+          width="fit"
+          className="!py-4 mx-auto text-lg !px-28 mt-7"
+        >
+          Launch app
+        </ButtonLink>
+        <div className="relative max-w-4xl mx-auto mt-16">
+          <Link passHref href={STACKLY_APP_URL}>
+            <Image
+              className="mx-auto border shadow-xl hover:shadow-2xl rounded-2xl border-surface-50"
+              alt="amount widget"
+              src="/assets/images/landing-amount-widget.png"
+              height={200}
+              width={512}
+            />
+          </Link>
+          <div className="absolute w-full -top-36 -z-10 h-[460px] bg-radial-gradient"></div>
         </div>
-      </div>
+        <div className="mx-auto flex items-center px-5 py-2 bg-black/5 rounded-[20px] w-fit space-x-6 mt-20">
+          <div className="flex items-center space-x-2">
+            <BodyText size={3} weight="medium" className="text-em-med">
+              Total Stacks created:
+            </BodyText>
+            <BodyText size={3} weight="bold">
+              732
+            </BodyText>
+          </div>
+          <div className="flex items-center space-x-2">
+            <BodyText size={3} weight="medium" className="text-em-med">
+              Total transactions:
+            </BodyText>
+            <BodyText size={3} weight="bold">
+              1232
+            </BodyText>
+          </div>
+        </div>
+      </section>
       <div className="h-[1px] bg-gray-100 mt-10"></div>
+      <section className="max-w-6xl px-6 mx-auto">
```

Comment:
Margin top should be in the section, right? The section wants to define the spacing from the sibling sections. Also, I think this should be padding, as we are going to have sections with different background colors and the spacing must have the same background color as the section. Also using max-w and mx-auto will prevent the background from filling the entire space. I think sections should be 100% width, without margin and with some background color, and a child div to define spacing and margins. Maybe in this way we don't need to define a margin to the section.

Reply:
In this particular section, the color is fine but for the hero it's not. Other sections will have different backgrounds too.

Reply:
fixed